packages/config/config/devices/0x0441/ibt4zwave.json

```
@@ -0,0 +1,269 @@
+	"paramInformation": {
+		"30[0xff000000]": {
+			"label": "Alarm Configuration - 1st Slot Notification Type",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 22,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+								{
+					"label": "Smoke Alarm",
+					"value": 1
+				},
+				{
+					"label": "CO Alarm",
+					"value": 2
+				},
+				{
+					"label": "CO2 Alarm",
+					"value": 3
+				},
+				{
+					"label": "Heat Alarm",
+					"value": 4
+				},
+				{
+					"label": "Water Alarm",
+					"value": 5
+				},
+				{
+					"label": "Access Control",
+					"value": 6
+				},
+				{
+					"label": "Home Security",
+					"value": 7
+				},
+				{
+					"label": "Power Management",
+					"value": 8
+				},
+				{
+					"label": "System",
+					"value": 9
+				},
+				{
+					"label": "Emergency Alarm",
+					"value": 10
+				},
+				{
+					"label": "Clock",
+					"value": 11
+				},
+				{
+					"label": "Appliance",
+					"value": 12
+				},
+				{
+					"label": "Home Health",
+					"value": 13
+				},
+				{
+					"label": "Siren",
+					"value": 14
+				},
+				{
+					"label": "Water Valve",
+					"value": 15
+				},
+				{
+					"label": "Weather Alarm",
+					"value": 16
+				},
+				{
+					"label": "Irrigation",
+					"value": 17
+				},
+				{
+					"label": "Gas Alarm",
+					"value": 18
+				},
+				{
+					"label": "Pest Control",
+					"value": 19
+				},
+				{
+					"label": "Light Sensor",
+					"value": 20
+				},
+				{
+					"label": "Water Quality Monitoring",
+					"value": 21
+				},
+				{
+					"label": "Home Monitoring",
+					"value": 22
+				}
+			]
+		},
+    "30[0xff0000]": {
+			"label": "Alarm Configuration - 1st Slot Notification Event",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Any Notification,",
+					"value": 255
+				},
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+        "30[0xff00]": {
+			"label": "Alarm Configuration - 1st Slot Notification Event Parameter",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+		},
+        "30[0xff]": {
+			"label": "Alarm Configuration - 1st Slot Action",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "No Action",
+					"value": 0
+				},
+				{
+					"label": "Open",
+					"value": 1
+				},
+				{
+					"label": "Close",
+					"value": 2
+				}
+			]
+		},
+		"31[0xff000000]": {
+			"$import": "#30[0xff000000]",
```

```suggestion
			"$import": "#paramInformation/30[0xff000000]",
```
same for the other ones.